requirements.txt

```
@@ -1,6 +1,8 @@
 Jinja2>=2.11.3
 numpy>=1.14.1
```

While you are at it, I think we can remove the version. I doubt we can run with such ancient versions ...
```suggestion
numpy
```